app/renderer/components/Inspector/Inspector.js

```diff
@@ -186,14 +188,47 @@ export default class Inspector extends Component {
            screenshotInteractionMode, isFindingElementsTimes, visibleCommandMethod,
            selectedInteractionMode, selectInteractionMode, selectAppMode, setVisibleCommandResult,
            showKeepAlivePrompt, keepSessionAlive, sourceXML, t, visibleCommandResult,
-           mjpegScreenshotUrl, isAwaitingMjpegStream} = this.props;
+           mjpegScreenshotUrl, isAwaitingMjpegStream, toggleCentroidMode, showCentroids} = this.props;
     const {path} = selectedElement;
 
     const showScreenshot = ((screenshot && !screenshotError) ||
                             (mjpegScreenshotUrl && !isAwaitingMjpegStream));
 
+    let screenShotControls = <div className={InspectorStyles['screenshot-controls']}>
+      <div className={InspectorStyles['action-controls']}>
+        <Tooltip title={t(showCentroids ? 'Hide Element Handles' : 'Show Element Handles')} placement="topRight">
```

Comment:
Hey @mykola-mokhnach , thanks for the feedback, much appreciated! I was just wondering if I would also need to send a PR to appium desktop to accomplish this?

<img width="832" alt="image" src="https://user-images.githubusercontent.com/85202734/178171591-0ccf2af5-a20e-4841-9853-b8cf9f626d06.png">

Reply:
unfortunately, yes, they share the same localization project. so yeah add them to the english json in that project, we'll merge it, then you can run the script as mentioned in the instructions you found so that the changes become part of this PR